plugins/source_testing.go

```diff
@@ -17,15 +16,8 @@ type ResourceTestCase struct {
 	ParallelFetchingLimit uint64
 	// SkipIgnoreInTest flag which detects if schema.Table or schema.Column should be ignored
 	SkipIgnoreInTest bool
-	// Verifiers are map from resource name to its verifiers.
-	// If no verifiers specified for resource (resource name is not in key set of map),
-	// non emptiness check of all columns in table and its relations will be performed.
-	Verifiers map[string][]Verifier
```

Comment:
I guess the removal of Verifiers is unrelated to pgx?

Reply:
I think it is related, because old verification was tied to pgx